Lab2.ipynb

```diff
@@ -0,0 +1,830 @@
+      "text/plain": [
+       "<Figure size 720x504 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "n = 200\n",
+    "left = 0\n",
+    "right = dist_200.size\n",
+    "cx = np.linspace(left, right, dist_200.size)\n",
+    "ave_x = cx/n\n",
+    "\n",
+    "plt.plot(ave_x, dist_200);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Q1 D\n",
+    "I chose the hypothetical number of dyas N to be 50. And I am seeing YxN = 100 gamma rays from my sourse.\n",
```

Comment:
I think you chose your Y to be 20. If N is 50 and we are assuming your observation is Y*N gamma rays, then you should be seeing Y*N = 20*50 = 1000 gamma rays.